app/Http/Controllers/BiosController.php

```diff
@@ -26,22 +25,16 @@ public function create()
         ]);
     }
 
-    public function store()
+    public function store(SaveBioRequest $request)
     {
-        // @todo: Why is this here? Why aren't we validating like we do everywhere else?
-        $form = CreateBioForm::fillOut(request()->input(), auth()->user());
+        $validatedData = $request->validated();
 
-        try {
-            $bio = $form->complete();
-        } catch (ValidationException $e) {
-            return redirect('bios/create')
-                ->withErrors($e->errors())
-                ->withInput();
-        }
+        $bio = Bio::create(array_merge($validatedData, [
+                'user_id' => Auth::id(),
+            ])
+        );
 
-        Session::flash('success-message', 'Successfully created new bio.');
-
-        return redirect('/bios/' . $bio->id);
+        return redirect('/bios/' . $bio->id)->with('success-message', 'Successfully created new bio.');
```

Comment:
This is a nice improvement over the session flash but lets also update it to use the named route.  You can use `return to_route('bios.show', $bio->id)`